### PR TITLE
Pill trilogy: The sweet return of the successor unchained

### PR DESCRIPTION
**Pull request by Dimasw99**
## About The Pull Request

The shipfreeze is no more and I fixed up the conversion.

This is an attempt to bring back the pill with its issues fixed while trying to keep its concept intact.

Ship to ship: A holopad and a wideband channel is now available
Player count: Default prisoner count is 6
Ship limit: The Sugarcube is limited to 2 ships per round

sugarcube

## Why It's Good For The Game

The pill design philosophy unfolded shiptests potential and it was quite evident on the results. This ship should still provide the unforgiving feeling of no mistakes allowed. It should be great fun

## Changelog
:cl:
add: Added Sugarcube Prison bus ship. The pill returns.
/:cl:

### CODE REVIEW

**Comment by Zevotech** [comment]
Woah Its The Bus From Black Ops Zombies

**Comment by Latentish** [comment]
Telecomms to start with seems like a bit much for a pill successor, don't you think?

**Comment by Dimasw99** [comment]
> 

It is a singular relay and not a 1 in all. It does not provide you with coms and is just a useless machine draining power

**Comment by swifterjack** [comment]
> > 
> 
> It is a singular relay and not a 1 in all. It does not provide you with coms and is just a useless machine draining power

holy shit based as hell, more ships need machines that solely exist to make the power econ harder

**Comment by Ms-Mee** [comment]
> > > 
> > 
> > It is a singular relay and not a 1 in all. It does not provide you with coms and is just a useless machine draining power
> 
> holy shit based as hell, more ships need machines that solely exist to make the power econ harder

That'd require power to actually be something that matters

**Comment by Spyroshark** [comment]
after playing a round with a full crew, i have to say that 6 people is WAY TOO MANY FOR A SHIP OF THIS SIZE. Having a single 3x6 room for 6 different people doing different things is a horrible experience, although considering this IS a successor to the pill, maybe it should be that way.

Complementary image from a two hour Sugarcube round:
hell

**Comment by Dimasw99** [comment]
> after playing a round with a full crew, i have to say that 6 people is WAY TOO MANY FOR A SHIP OF THIS SIZE. Having a single 3x6 room for 6 different people doing different things is a horrible experience, although considering this IS a successor to the pill, maybe it should be that way.

Yeah the overwhelming crew size is intial because the sugarcube is intended to be expanded but also offer plenty of respawns in a ship that is prone to deaths. The previous number used to be 3 for a pill but if people think that it is too much I may turn it down to 5.

In my play through with the sugarcube we were able to grab some EVA gear from some legions, expand the ship twice the size, get RnD going and had the true zero to hero pill experience. Likewise I may add one more pickaxe and a singular soft suit. Lastly ship to ship interaction for trade is also always possible for EVA gear.

File: _maps/configs/sugarcube.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"map_name": "Sugarcube-Class Prison Bus Ship",
+	"map_short_name": "Sugarcube-class",
+	"map_path": "_maps/shuttles/shiptest/sugarcube.dmm",
+	"map_id": "sugarcube",
+	"limit": 2,
+	"job_slots": {
+		"Prisoner": 6
```
Comment (ZephyrTFA) [inline _maps/configs/sugarcube.json]:
```suggestion
		"Prisoner": 4
```
6 is still just a bit too much for a ship this small; if they want more crew they can open more slots

**Comment by Dimasw99** [comment]
The precharger now starts with 20% charged instead of full
Added two beers so that dwarf players don't have to sweat power game speedrun fermentation
Added a poster
Added 3 floorpills. Enjoy

**Pull request merged**